lib/src/widgets/credit_card.dart

```diff
@@ -40,12 +40,17 @@ class _CreditCardState extends State<CreditCard> {
 
   bool _tokenizeCard = false;
 
+  bool _manualPayment = false;
+
   @override
   void initState() {
     super.initState();
     setState(() {
       _tokenizeCard = widget.config.creditCard?.saveCard ?? false;
     });
+    setState(() {
+      _manualPayment = widget.config.creditCard?.manual ?? false;
+    });
```

Comment:
Can you please merge setting the state in one block?

```dart
    setState(() {
      _tokenizeCard = widget.config.creditCard?.saveCard ?? false;
      _manualPayment = widget.config.creditCard?.manual ?? false;
    });
```

Reply:
sure 👍🏻